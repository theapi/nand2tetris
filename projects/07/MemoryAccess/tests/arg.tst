
load arg.asm,
output-file arg.out,
compare-to arg.cmp,
output-list RAM[401]%D1.6.1 RAM[402]%D1.6.1;


repeat 75 {
  ticktock;
}

output;
