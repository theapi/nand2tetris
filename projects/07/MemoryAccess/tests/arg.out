|RAM[401]|RAM[402]|
|     21 |     22 |
